### PR TITLE
fix/past-elections#4090 last

### PR DESCRIPTION
**Pull request by GoldWolf115**
fix/past-elections#4090

### CODE REVIEW

File: packages/ui/.env.example [hunk 1]
```diff
@@ -28,4 +28,4 @@ REACT_APP_IMAGE_REPORT_FORM_URL="https://airtable.com/formXYZ?prefill_image={ima
 # Development
 REACT_APP_API_BENCHMARK=false
 REACT_APP_AVATAR_UPLOAD_URL=https://atlas-services.joystream.org/avatars
-REACT_APP_IS_UNDER_MAINTENANCE=true
+REACT_APP_IS_UNDER_MAINTENANCE=true
```
Comment (chrlschwb) [inline packages/ui/.env.example:31]: please uncommit changes to this file

**Comment by thesan** [comment]
@chrlschwb please set `jsg-code-review` only after you have approved a PR

File: packages/ui/src/council/components/election/pastElection/PastElectionsList/PastElectionsListRow.tsx [hunk 2]
```diff
@@ -11,7 +11,6 @@ import { Fraction } from '@/common/components/typography/Fraction'
 import { ElectionRoutes } from '@/council/constants'
 import { PastElection } from '@/council/types/PastElection'
 import { CountInfo, Info } from '@/memberships/components/MemberListItem/Fields'
-
```
Comment (chrlschwb) [inline packages/ui/src/council/components/election/pastElection/PastElectionsList/PastElectionsListRow.tsx:14]: please uncommit this change

File: packages/ui/.env.example [hunk 3]
```diff
@@ -1,31 +0,0 @@
...
-IMAGE_SAFETY_BLACKLIST_URL="https://api.airtable.com/v0/appXYZ/tableXYZ?api_key=keyXYZ&filterByFormula={blacklisted}"
-IMAGE_SAFETY_BLACKLIST_JSON_PATH=image
-
-## Reporting hyperlink
-REACT_APP_IMAGE_REPORT_FORM_URL="https://airtable.com/formXYZ?prefill_image={image}&prefill_page={context}&hide_image=true&hide_page=true"
-
-# Development
-REACT_APP_API_BENCHMARK=false
-REACT_APP_AVATAR_UPLOAD_URL=https://atlas-services.joystream.org/avatars
-REACT_APP_IS_UNDER_MAINTENANCE=true
```
Comment (chrlschwb) [inline packages/ui/.env.example:31]: please uncommit the deletion of this file

**Review by chrlschwb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**